Resources/public/sonata-dashboard.front.css

```diff
@@ -79,15 +79,6 @@
   /*border: 1px solid #000000;*/
   background-color: rgba(255, 150, 0, 0.5); }
 
-/*# sourceMappingURL=dashboard.css.map */
-
-{
-"version": 3,
-"mappings": ";AACA,yBAA0B;EACtB,MAAM,EAAE,IAAI;;AAEhB,+BAAgC;EAC5B,KAAK,EAAE,OAAO;;AAGlB,mBAAoB;EAChB,UAAU,EAAE,MAAM;;AAGtB,kCAAmC;EAC/B,UAAU,EAAE,OAAO;;;AAIvB,yBAA0B;EACtB,QAAQ,EAAE,QAAQ;EAClB,UAAU,EAAE,IAAI;;;AAIpB,qCAAsC;EAClC,gBAAgB,EAAE,sBAAsB;;;;AAQ5C,qCAAsC;EAClC,OAAO,EAAE,YAAY;EACrB,OAAO,EAAE,GAAG;EACZ,SAAS,EAAE,IAAI;EACf,KAAK,EAAE,OAAO;;AAElB,6DAA8D;EAC1D,OAAO,EAAE,KAAK;EACd,KAAK,EAAE,IAAI;EACX,gBAAgB,EAAE,OAAmB;;AAEzC,yDAA0D;EACtD,OAAO,EAAE,KAAK;EACd,gBAAgB,EAAE,OAAkB;;;AAGxC,kFAAmF;EAC/E,OAAO,EAAE,KAAK;;AAElB,6EAA8E;EAC1E,OAAO,EAAE,KAAK;;;AAIlB,6BAA8B;EAC1B,WAAW,EAAE,IAAI;EACjB,MAAM,EAAE,kBAAiC;;;;;;AAQ7C,qCAAsC;EAClC,gBAAgB,EAAE,wBAAwB;EAC1C,UAAU,EAAE,IAAI;;;AAIpB,sCAAuC;EACnC,MAAM,EAAE,IAAI;;;AAIhB,+DAAgE;;EAE5D,gBAAgB,EAAE,sBAAsB",
-"sources": ["../dashboard.scss"],
-"names": [],
-"file": "dashboard.css"
-}
```

Comment:
Isn't that already done in your other PR ? If yes please rebase, it will make the diff more simple.